code/datums/wires/explosive.dm

```diff
@@ -1,5 +1,5 @@
 /datum/wires/explosive/New(atom/holder)
-	add_duds(2) // In this case duds actually explode.
+	add_duds(1) // In this case duds actually explode.

```

Comment:
guessing this has some unintended side effects. you might want to put the number of duds on the wires/explosive datum instead then have the chem_grenade overwrite said variable to 1.